tests/smoke-npm-remove-transitive.yaml

```diff
@@ -0,0 +1,174 @@
+input:
+    job:
+        package-manager: npm_and_yarn
+        allowed-updates:
+            - dependency-type: direct
+              update-type: all
+            - dependency-type: indirect
+              update-type: security
+        dependencies:
+            - '@dependabot-fixtures/npm-transitive-dependency'
+        experiments:
+            npm-transitive-dependency-removal: true
+            npm-transitive-security-updates: true
+        ignore-conditions:
```

Comment:
Ok, I can do that. The CLI automatically generates these to make the tests less likely to fail, but since we control this source it seems like no one will bump it.

Reply:
Ah, makes sense. However, for security udpates we typically expect an update to the lowest fix version so adding in these ignore conditions would affect that logic.